awx_collection/plugins/modules/tower_workflow_job_template_node.py

```diff
@@ -93,6 +93,24 @@
         - Can be a job template, project, inventory source, etc.
         - Omit if creating an approval node (not yet implemented).
       type: str
+    approval_node:
```

Comment:
Should approval node and unifited_job_template be mutually exclusive? Or can you do both at the same time?

Reply:
Mutually exclusive, I did not test invoking both, but in theory the node would be created then converted to approval.